pkg/api/types.go

```diff
@@ -195,6 +195,119 @@ type VolumeSource struct {
 	NFS *NFSVolumeSource `json:"nfs"`
 }
 
+// Similar to VolumeSource but meant for the administrator who creates PVs.
+// Exactly one of its members must be set.
+type PersistentVolumeSource struct {
+	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// HostPath represents a directory on the host.
+	// This is useful for development and testing only.
+	// on-host storage is not supported in any way
+	HostPath *HostPathVolumeSource `json:"hostPath"`
+}
+
+type PersistentVolume struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	//Spec defines a persistent volume owned by the cluster
```

Comment:
I would not mind a discussion around this.  There is nothing that requires PV to have no namespace except that we've been thinking of them as analogs to nodes.

It would be entirely possible for an admin to provision a PV _in a specific namespace_.  Maybe you get the fancy 1TB volume for your data warehouse probject.  Then again, this was thought to be achievable using @derekwaynecarr's quota framework.